frontend/src/components/common/Resource/index.test.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import * as AllComps from '.';
+
+const avoidCheck = [
+  '.stories',
+  '.test',
+  'index',
+  '__snapshots__',
+  // Not exported on purpose
+  'ReleaseNotes',
+  'ActionsNotifier',
+  'AlertNotification',
+  'ConfirmButton',
+  'NamespacesAutocomplete',
```

Comment:
We could export these 2 as well IMO.

Reply:
I noticed that these are missing from their test in the common directory, I have moved them to be included there. Unless it would be better fitting for them to be in the Resource test then I shall move them there.
I figured since it would be testing for the export where it exist that would be the go to move?